carbon/cli.py

```diff
@@ -55,4 +62,9 @@ def get_args():
         default=os.getcwd(),
         help="Specify folder where image should be stored (defaults to current directory)",
     )
+    parser.add_argument(
+        "--driver-path",
+        help="Path to the executable, if it is not given it reads value from environment variable (DRIVER_PATH)",
+        **environ_or_required("DRIVER_PATH"),
```

Comment:
nice, so if set in environment this argument is optional, if not set it becomes required, right?

Reply:
Yes, that's exactly right!

Reply:
By the way, click and other command-line applications provide this out-of-box, but we have to do this here.

Reply:
Good to know :)
Thanks